frontend/src/actions/auth.js

```diff
@@ -101,6 +101,34 @@ export function resendConfirmationCode (email) {
   }
 }
 
+export const RESET_CODE_REQUEST = 'RESET_CODE_REQUEST'
```

Comment:
Can we have consistent names for the actions?

Reply:
like FORGOT_CODE_REQUEST ?